README.md

```diff
@@ -8,35 +8,35 @@ The `master` branch only contains the `SimpleSwap` contract for now. Everything
 
 ## Tests
 
-This is a regular truffle project. You can either use `truffle test` (if installed) or `npm test` which will use a locally installed truffle.
+This is a hardhat project using the truffle plugin (for tests as this used to be truffle-based). You can run tests using `yarn test`.
 
 ```sh
-npm install
-npm test
+yarn
```

Comment:
With node 14, I encounter the following problem:

![image](https://user-images.githubusercontent.com/33000441/111456003-c2133100-8716-11eb-9926-47c350db59d7.png)


With node 12, everything is ok. I would suggest to mention that we require node 12.

Reply:
fixed now